sim/visuals/microbit.ts

```diff
@@ -1494,19 +1512,19 @@ path.sim-board {
         attachButtonEvents(stateButton: Button, buttonOuter: SVGElement, elButton: SVGElement) {
             let pressedTime: number;
             pointerEvents.down.forEach(evid => buttonOuter.addEventListener(evid, ev => {
-                // console.log(`down ${stateButton.id}`)
                 stateButton.pressed = true;
-                svg.fill(elButton, this.props.theme.buttonDown);
+                this.updateButtonPairs();
```

Comment:
These lines of code are not equivalent. The deleted line includes updating the touch logo, the code in `updateButtonPairs` does not. This means that pointer events are no longer working for the touch logo. Note that the `on logo` block needs to be added to the user program for the logo to become interactive.

Reply:
Fixed here [2c9f49e](https://github.com/microbit-matt-hillsdon/pxt-microbit/pull/5/commits/2c9f49e32b4c3a56720d2fba56d4079f55352553)